scout/server/blueprints/variants/views.py

```diff
@@ -293,7 +296,14 @@ def cancer_variants(institute_id, case_name):
     else:
         page = int(request.args.get("page", 1))
         form = CancerFiltersForm(request.args)
-        form.chrom.data = request.args.get("chrom", None)
+        # set chromosome to all chromosomes
+        form.chrom.data = request.args.get("chrom", "")
+        # set gene panels to case-specific gene panels
```

Comment:
```suggestion
        # set gene panels to case-specific default gene panels
```

Reply:
Or just remove the comments alltogether - or change them to discuss case level versus institute level or sth, but as they stand they only make me confused.. 😅

Reply:
Ok I'll remove them